ruby/vampires.rb

```diff
@@ -1,15 +1,78 @@
+puts "How many employees are there to be processed?"
+employees = gets.to_i
+
+while employees > 0
```

Comment:
Make sure that you are indenting everything nested under your while for readability.  I use https://github.com/airbnb/ruby as a style guide when Im wondering how to format my code.